### PR TITLE
Hi taylor

### PR DESCRIPTION
**Pull request by achansler**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -12,47 +12,27 @@ $('a.designSprintFullStudyReadLink').on('click', function() {
 	$( 'div.designSprintFullStudy' ).toggleClass( 'show' );
```
Comment (achansler) [inline app.js:12]: @talitore 
Hi! So I'm having some problems over here and I'm not sure of the best way to approach this. What I'd like to happen is when I click on the "Read" link, the hidden content which is all of the project info expands on the page and the "Read" link then says "Close". Right now all of the links change to say "Close" and never change back to "Read". 
Basically I want it to act as an accordion, but I can't seem to make it work. What do you suggest?

Comment (talitore) [reply app.js:12]: Hi!!!! So the problem is actually on lines 1-4 above. Your `$('a.read')` selector means "all anchors with class of 'read'". So if you replace that with all instances of `$('a.read')` in those 4 lines it reads conceptually like "when clicking any anchor with the class of 'read' do the following: set the text of all anchors with class of 'read' to 'Close' and then add the class of 'close' to all anchors with class of 'read'".

TLDR; clicking any anchor is then doing stuff to all anchors not just the one you click.

To change that behavior you'd want to change the inner selectors on lines 2-3 (i.e. the `$('a.read')` bit) so that they only manipulate the link clicked. To help with this Javascript has a `this` keyword which is a reference to the object in scope. If you were to reference `this` instead of `$('a.read')` it would only fire on the anchor clicked. Something like this is what I'm referring to:

```
$('a.read').on('click', function() {
	this.text('Close');
	this.addClass('close');
})
```

UNFORTUNATELY this won't work because `this` is referring to a vanilla Javascript object and you're wanting to call jQuery methods (`text() and addClass()`. So you'd have to convert `this` to a jQuery object by just wrapping it in the standard `$()` notation a la `$(this)`. That'd turn lines 1-4 into:

```
$('a.read').on('click', function() {
	$(this).text('Close');
	$(this).addClass('close');
})
```

Or since you can chain jQuery methods you can just shorten it to the following if you wanted.

```
$('a.read').on('click', function() {
	$(this).text('Close').addClass('close');
})
```

Regarding the link not changing back to Read: take a look at line 7 :D
